.builds/linux.yml

```diff
@@ -7,9 +7,15 @@ tasks:
   - stable: |
       cd vte
       $HOME/.cargo/bin/cargo +stable test
+      $HOME/.cargo/bin/cargo +stable test --features=ansi
+      $HOME/.cargo/bin/cargo +stable test --features=ansi --no-default-features
+      $HOME/.cargo/bin/cargo +stable test --features=ansi,no_alloc
```

Comment:
I see little point of having so many different test variations. Without ansi shouldn't make any difference when things work with it.

Reply:
Fixed in 1b51612